.github/workflows/pytest.yml

```diff
@@ -24,6 +24,12 @@ jobs:
       GOOGLE_API_KEY: ${{ secrets.GOOGLE_API_KEY }}
       GOOGLE_CSE_ID: ${{ secrets.GOOGLE_CSE_ID }}
 
+      # Azure OpenAI env vars
+      AZURE_API_KEY: ${{ secrets.AZURE_API_KEY }}
+      AZURE_OPENAI_API_BASE: ${{ secrets.AZURE_OPENAI_API_BASE }}
+      AZURE_OPENAI_API_VERSION: "2023-05-15"
```

Comment:
what about the `AZURE_GPT_MODEL_NAME`

Reply:
I missed this, let me know when I can add it

Reply:
I will add it